directives/questionnaire.py

```diff
@@ -387,7 +395,6 @@ class FreeText(QuestionMixin, Directive):
         'no-standard-prompt': directives.flag,
         'shorter-prompt': directives.flag,
         'class': directives.class_option,
-        'required': directives.flag,
```

Comment:
I don't think you can remove the `required` option here. This dict does not inherit anything from the super class QuestionMixin since the value is overridden here. It would make sense to inherit the common options, but it would require code like this:

```
option_spec = dict(QuestionMixin.option_spec)
option_spec.update({ .... }) # new key-value pairs
```

Reply:
`required` option has been written twice in that option dict, so there still would be one of them.